02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+
+const { getPeople, addPerson, updatePeople, deletePeople } = require("../controllers/people");
+
+router.get('/', getPeople)
+router.post('/post', addPerson)
```

Comment:
Nit: Typically the get and post routes will be the same.  So for the post you can use `router.post('/', addPerson)`.  This is a good practice because you want to get and post to the same underlying collection (people).